src/ncbitaxon.py

```diff
@@ -229,18 +235,17 @@ def convert(taxdmp_path, output_path, taxa=None):
 .
 """
             )
-        for label, parent in predicates.items():
-            predicate = label_to_id(label)
+
+        for label, (parent, curie) in predicates.items():
             parent = parent.replace("oboInOwl", "oio")
-            output.write(
-                f"""
-ncbitaxon:{predicate} a owl:AnnotationProperty
-; rdfs:label "{label}"^^xsd:string
-; oboInOwl:hasScope "{parent}"^^xsd:string
-; rdfs:subPropertyOf oboInOwl:SynonymTypeProperty
-.
-"""
-            )
+            if curie is None:
+                curie = f"ncbitaxon:{label_to_id(label)}"
+            output.write(dedent(f"""
+                {curie} a owl:AnnotationProperty ;
+                    rdfs:label "{label}"^^xsd:string ;
```

Comment:
In the case of OMO annotation, here is adding different label, not the same as the source annotation.

Reply:
Here's an example: OMO:0003000 label is abbreviation
```
OMO:0003000 a owl:AnnotationProperty ;
    rdfs:label "acronym"^^xsd:string ;
    oboInOwl:hasScope "oio:hasBroadSynonym"^^xsd:string ;
    rdfs:subPropertyOf oboInOwl:SynonymTypeProperty .
```

Reply:
Addressed in 4d94d13